themes/base/accordion.css

```diff
@@ -21,3 +21,14 @@
 	border-top: 0;
 	overflow: auto;
 }
+
+/* RTL support */
+.ui-accordion-rtl {
+	direction: rtl;
+}
+.ui-accordion-rtl .ui-accordion .ui-accordion-header {
+	padding: .5em .7em .5em .5em;
+}
+.ui-accordion-header-iconRtl.ui-icon {
+	transform: scaleX(-1);
```

Comment:
The support aims to make the changes required to mirror the app as minimal as possible (just set `dir='rtl'`).
So I think it is better for the user to write his app once & just change one attribute (`dir`) to get his app mirrored.